ci-operator/step-registry/monitoring-plugin/tests/perses-ui-ivt/monitoring-plugin-tests-perses-ui-ivt-commands.sh

```
@@ -0,0 +1,142 @@
+fi
+
+kubeadmin_password=$(cat "${KUBEADMIN_PASSWORD_FILE}")
+echo "Successfully read kubeadmin password from ${KUBEADMIN_PASSWORD_FILE}"
+
+# Set proxy vars.
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+  source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+## skip all tests when console is not installed.
+if ! (oc get clusteroperator console --kubeconfig=${KUBECONFIG}) ; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)